jwala-common/src/main/java/com/cerner/jwala/common/domain/model/webserver/WebServer.java

```diff
@@ -206,4 +208,12 @@ public String toString() {
                 ", state=" + state +
                 '}';
     }
+
+    private String getGroupNames(Collection<Group> groups) {
+        StringBuilder groupsNames = new StringBuilder();
+        for (Group group : groups) {
+            groupsNames.append(group.getName()).append(" ");
```

Comment:
Can you add a comma separator, if the group name has a space in it you cannot tokenize?